avalon/tools/cbsceneinventory/model.py

```diff
@@ -49,10 +49,13 @@ def data(self, index, role):
         # Add icons
         if role == QtCore.Qt.DecorationRole:
             if index.column() == 0:
+                # Override color
+                node = index.internalPointer()
```

Comment:
You don't need this line right? The variable is already defined above?

Reply:
I will refactor it out together with [this line](https://github.com/aardschok/core/blob/bc129635912c4fc1185947abb441ffd540333bbc/avalon/tools/cbsceneinventory/model.py#L64)

Reply:
If you mean [this line](https://github.com/aardschok/core/blob/bc129635912c4fc1185947abb441ffd540333bbc/avalon/tools/cbsceneinventory/model.py#L65), yes!